ksql-api/src/main/java/io/confluent/ksql/api/server/BaseSubscriber.java

```diff
@@ -47,26 +51,26 @@
    *                be executed on this context. This ensures the code is never executed
    *                concurrently by more than one thread.
    */
-  public ReactiveSubscriber(final Context context) {
+  public BaseSubscriber(final Context context) {
```

Comment:
nit: validate params that will be stored in object state; ensuring object does not get into an invalid state.

Reply:
ack